cmd/virtual_machine_init.go

```diff
@@ -68,7 +68,7 @@ func (iva *initVMAction) run() error {
 
 	err = dependency.InstallOptionalDeps(iva.optionalDepGroups, iva.logger)
 	if err != nil {
-		iva.logger.Error(fmt.Sprintf("Dependency error: %s", err))
+		iva.logger.Errorf("Dependency error: %s", err.Error())
```

Comment:
Could we use `%v` and `err` instead of `%s` and `err.Error()` to be consistent with other logging statements?

Reply:
Thanks ! I fixed.